src/Symfony/Component/Yaml/Yaml.php

```diff
@@ -21,10 +21,11 @@
 class Yaml
 {
     /**
-     * Parses YAML into a PHP array.
+     * Parses YAML into a PHP array or an stdClass object.
```

Comment:
Updated. I also removed the sentence below this one as it was basically saying the same thing.